### PR TITLE
Indexed db extension

### PR DESCRIPTION
**Pull request by ffghdfh**
add an IndexedDB extension to the website

### CODE REVIEW

**Comment by ffghdfh** [comment]
HAHA! I PASSED THE CHECKS!

File: extensions/ffghdfh/indexedDB.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+                defaultValue: "myKey",
+              },
+            },
+          },
+        ],
+      };
+    }
+
+    setDBName(args) {
+      const { NAME } = args;
```
Comment (LilyMakesThings) [inline extensions/ffghdfh/indexedDB.js:192]: You could do this in the function argument as opposed to setting it retroactively, if you wanted to

File: extensions/ffghdfh/indexedDB.js [hunk 2]
```diff
@@ -0,0 +1,222 @@
+// Name: Indexed Database
```
Comment (LilyMakesThings) [inline extensions/ffghdfh/indexedDB.js:1]: Credit yourself here! You made the extension! :)

**Review by LilyMakesThings — APPROVED** [review]

File: extensions/ffghdfh/indexedDB.js (hunk 1 above)
Comment (ffghdfh) [reply extensions/ffghdfh/indexedDB.js:192]: uhh i'm sorry, but i'm not quite sure what you mean by that 😅

File: extensions/ffghdfh/indexedDB.js (hunk 2 above)
Comment (ffghdfh) [reply extensions/ffghdfh/indexedDB.js:1]: done! i added my discord username!

**Review by ffghdfh — COMMENTED** [review]
i added my discord thingy to the description!

**Review by ffghdfh — COMMENTED** [review]
i forgot a punctuation thing at the end of the description, added that as well!

File: extensions/ffghdfh/indexedDB.js (hunk 1 above)
Comment (ffghdfh) [reply extensions/ffghdfh/indexedDB.js:192]: oh i got it, one sec imma modify it

**Comment by Thebloxers998** [comment]
Why was this closed

**Comment by yuri-kiss** [comment]
> Why was this closed

they reopened a new pr a long time ago :skull: please check before you comment

**Comment by Thebloxers998** [comment]
> > Why was this closed
> 
> they reopened a new pr a long time ago 💀 please check before you comment 


 how did I not notice that  💀

**Pull request closed without merging**